package.json

```
@@ -43,8 +43,11 @@
     "test:e2e": "pnpm run release:build && cd test/e2e && pnpm run start",
     "test:e2e-website": "npx playwright test test/e2e-website --config test/e2e-website/playwright.config.ts",
     "test:e2e-website:dev": "PLAYWRIGHT_TEST_BASE_URL=http://localhost:3001 npx playwright test test/e2e-website --config test/e2e-website/playwright.config.ts",
-    "test:regressions": "pnpm run release:build && cd test/regressions && pnpm run start",
-    "test:regressions:dev": "cd test/regressions && pnpm run start",
+    "test:regressions": "cross-env NODE_ENV=production pnpm test:regressions:build && concurrently --success first --kill-others \"pnpm test:regressions:run\" \"pnpm test:regressions:server\"",
+    "test:regressions:build": "cross-env TEST_BUILD=true vite build --config test/regressions/vite.config.js",
+    "test:regressions:dev": "pnpm test:regressions:build && pnpm test:regressions:server",
```

I would want to use this here:
```suggestion
    "test:regressions:dev": "cross-env TEST_BUILD=true vite --config test/regressions/vite.config.js --port 5001",
```

but Vite seems unable to build properly in dev mode compared to webpack, so falling back to this 🙃.